### ISSUE TITLE
the api reference is not complete

### ISSUE THREAD

**Issue by alopezq** [BALKANGraph/OrgChartJS#2]
Your api web page is not complete. Only 5 methods are described. 

https://balkangraph.com/OrgChartJS/Docs/APIReference

**Comment by BALKANGraph** [comment]
Yes, it is complete, currently we are writing the doc, will give you an update when it is ready

**Comment by bsalines91** [comment]
Its been 20 days and API doc references still point to no Dom element and provide no information about the data accepted for each one. Also, I believe "mouseScroolBehaviour" should be "mouseScrollBehaviour".
